app/assets/stylesheets/layout_components/reveal-modal.scss

```diff
@@ -51,7 +52,6 @@ dialog .close-reveal-modal {
 
   @include breakpoint(small only) {
     width: 100%;
-    top: 50px !important;
```

Comment:
@DaveAMA this was overriding the positioning for the chat on mobile, it needed an important because it was overriding the positioning on desktop. so once it had an important I was unable to override it. So I took a look and checked how our normal modals look on mobile without the top:50px, and honestly it wasn't providing any real value so I removed it that I was able to have my top override for the chat modal on mobile...if you have a better idea I am totally open to it. I really don't like all these important's but I don't see a way around it when we have to be using the base styles from the reveal-modal on our chat modal.